packages/inferno-from-react-generator/src/__tests__/react-test-cases/react-memo.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React  from 'react';
+import {memo}  from 'react';
```

Comment:
Minor comment.

Let's add spaces:
```suggestion
import { memo }  from 'react';
```